src/libraries/System.Private.CoreLib/src/System/Globalization/OrdinalCasing.Icu.cs

```diff
@@ -12,11 +12,11 @@ namespace System.Globalization
     internal static partial class OrdinalCasing
     {
         // s_noCasingPage means the Unicode page doesn't support any casing and no case translation is needed.
-        private static ushort [] s_noCasingPage = Array.Empty<ushort>();
+        private static readonly ushort[] s_noCasingPage = Array.Empty<ushort>();
 
         // s_basicLatin is covering the casing for the Basic Latin & C0 Controls range.
         // we are not lazy initializing this range because it is the most common used range and we'll cache it anyway very early.
-        private static ushort [] s_basicLatin =
+        private static readonly ushort[] s_basicLatin =
```

Comment:
This could/should be a property returning ROSpan<ushort> now that the language support exists, yes?

Reply:
No, it's stored into a ushort[]?[].